internal/connection/cluster.go

```diff
@@ -1,31 +1,13 @@
-/*
- * Copyright (c) 2008-2021, Hazelcast, Inc. All Rights Reserved.
- *
- * Licensed under the Apache License, Version 2.0 (the "License")
- * you may not use this file except in compliance with the License.
- * You may obtain a copy of the License at
- *
- * http://www.apache.org/licenses/LICENSE-2.0
- *
- * Unless required by applicable law or agreed to in writing, software
- * distributed under the License is distributed on an "AS IS" BASIS,
- * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
- * See the License for the specific language governing permissions and
- * limitations under the License.
- */
-package internal
+package connection
```

Comment:
Why was moving this code to a package was necessary?

Reply:
It sounds natural to move connection related stuff under the same package. Right now, there are two type connections which are REST to cluster and through Go Client.